doc/release/multilocale.rst

```diff
@@ -4,233 +4,198 @@
+
+.. _set-comm-conduit:
+
+Setting CHPL_COMM_SUBSTRATE
++++++++++++++++++++++++++++
+
+Advanced users can set ``CHPL_COMM_SUBSTRATE`` to indicate the GASNet
+conduit that they wish to use. Conduits are optional components in the
+GASNet library. GASNet uses different conduits to support different
+networks. is a Novice users can leave this unset and Chapel will make a
+choice for them.
+
+Most settings for ``CHPL_COMM_SUBSTRATE`` rely on the particular network
+hardware. The options include:
+
+ibv
+    OpenIB/OpenFabrics Verbs for InfiniBand
+    (see :ref:`Using Chapel with InfiniBand <readme-infiniband>`)
+mxm
+    Mellanox MXM for InfiniBand
+portals4
+    Portals4 (BETA) for the Portals 4.x API
+pami
+    PAMI for IBM Power 775, BlueGene/Q and others
+shmem
+    SHMEM for SGI Altix
+gemini
+    Gemini for Cray XE and XK series
+    (see :ref:`Using Chapel on Cray Systems <readme-cray>`)
+aries
+    Aries for Cray XC series
+    (see :ref:`Using Chapel on Cray Systems <readme-cray>`)
+ofi
+    OFI for multiple networks supported by libfabric
+psm
+    PSM for Intel's OmniPath fabric
+mpi
+    MPI - portable conduit, works on any network with MPI 1.1 or newer
+udp
+    UDP - portable conduit, works on any network with a TCP/IP stack
+    (see :ref:`Using the Portable UDP Conduit <using-udp>`)
 
-   .. code-block:: bash
+See the GASNet documentation for more information on each of these conduits.
 
-     chpl -o hello6-taskpar-dist $CHPL_HOME/examples/hello6-taskpar-dist.chpl
+Current defaults are:
 
+====================  ===================
+CHPL_TARGET_PLATFORM  CHPL_COMM_SUBSTRATE
+====================  ===================
+cray-cs                ibv
+cray-xc                aries
+cray-xe                gemini
+cray-xk                gemini
+cray-xt                mpi
+pwr5                   lapi
+pwr6                   ibv
+other                  udp
+====================  ===================
 
-#. When you compile a Chapel program for multiple locales, you should
-   typically see two binaries (e.g., hello6-taskpar-dist and
-   hello6-taskpar-dist_real).  The first binary contains code to
-   launch the Chapel program onto the compute nodes using a GASNet run
-   command.  The second contains the program code itself.  You can use
-   the ``-v`` flag to see the commands used to launch your program.  See
-   :ref:`readme-launcher` for further details.
+.. _set-comm-segment:
 
+Setting CHPL_GASNET_SEGMENT
++++++++++++++++++++++++++++
 
-#. Multi-locale executions require the number of locales to be
-   specified on the command line.  Other than this, execute your
-   Chapel program as usual.  For example:
+Advanced GASNet users can set ``CHPL_GASNET_SEGMENT`` to choose a memory
+segment to use with GASNet. A GASNet segment is a region of memory that
+is expected to be used for remote memory access. The GASNet library works
+to make memory in this segment is available for accelerated memory access
```

Comment:
There's a typo somewhere in "works to make memory in this segment is available", either a missing "sure" after "make", or a spurious "is".